test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,80 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+require('dotenv').config();
+
+describe('Should interact with the user', () => {
+  const urlBase = 'https://api.github.com/user/following';
+  const user = '/aperdomob';
+
+  describe('Should follow the user', () => {
+    let followUser;
+    before(async () => {
+      followUser = await axios.put(
+        `${urlBase}${user}`,
+        {},
+        {
+          headers: {
+            Authorization: `token ${process.env.ACCESS_TOKEN}`
+          }
+        }
+      );
+    });
+    it('Should verify that you follow the user', () => {
+      expect(followUser.status).to.equal(StatusCodes.NO_CONTENT);
+      expect(followUser.data).to.equal('');
+    });
+  });
+  describe('Should interactive with de user list', () => {
+    let response;
+    let loginUser;
+    before(async () => {
+      response = await axios.get(`${urlBase}`, {
+        headers: {
+          Authorization: `token ${process.env.ACCESS_TOKEN}`
+        }
+      });
+      loginUser = response.data.find(
+        (elemento) => elemento.login === 'aperdomob'
+      );
+    });
+    it('Should check the list and verify that follow the user', async () => {
+      expect(loginUser.login).to.equal('aperdomob');
+    });
+  });
+  describe('Following the user again', () => {
+    let followUserAgain;
+    before(async () => {
+      followUserAgain = await axios.put(
+        `${urlBase}${user}`,
+        {},
+        {
+          headers: {
+            Authorization: `token ${process.env.ACCESS_TOKEN}`
+          }
+        }
+      );
+    });
+    it('Should verifiy that the user is followed', async () => {
```

Comment:
this async is not needed. async-await usage is only required when calling APIs or performing actions that needs to resolve a promise, the expects are not promises, so, the async is not needed.